requirements.txt

```diff
@@ -1,16 +1,16 @@
 # Version updates managed by dependabot
 
 betelgeuse==1.10.0
-broker[docker]==0.3.2
-cryptography==41.0.0
-deepdiff==6.3.0
-dynaconf[vault]==3.1.12
+broker[docker]==0.3.3
```

Comment:
@ogajduse Can you update this PR to pull missed cherrypicks from requirements-optional.txt?